src/components/ValueInput/ValueInputHeader.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import * as React from 'react';
+import { CachedImage } from 'react-native-cached-image';
+import styled, { withTheme } from 'styled-components/native';
+import type { Option } from 'models/Selector';
+import { resolveAssetSource } from 'utils/textInput';
+import { images } from 'utils/images';
+import type { Theme } from 'models/Theme';
+import Icon from 'components/Icon';
+import { themedColors } from 'utils/themes';
+import { BaseText, MediumText } from 'components/Typography';
+import { fontStyles } from 'utils/variables';
+
+type Props = {
+  theme: Theme,
+  asset: Option,
+  labelText: string,
+  onLabelPress: () => void,
+  onAssetPress: () => void,
+};
+
+const Wrapper = styled.View`
+  width: 100%;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 15px;
+  height: 24px;
+`;
+
+const SideWrapper = styled.TouchableOpacity`
```

Comment:
This could use some paddings on sides (based on position (right / left)).
As if max value is super long, it touches with asset value. Or wise versa.

Or maybe we need to wrap / shorten token name to fit everything into one line?

Reply:
I'm not sure I follow - where should I add paddings?

Reply:
On the horizontal sides.
Try selecting asset with a long name and / or the one that has significant balance (that takes up lots of scree space)

Reply:
Added some padding between the asset and amount. Also asset name is ellipsized if necessary.

<img width="353" alt="Screenshot 2020-10-08 at 15 48 28" src="https://user-images.githubusercontent.com/6280457/95473360-51583380-0984-11eb-8307-4bba72140fcb.png">

Reply:
Perfect :)